src/store.js

```diff
@@ -108,7 +110,7 @@ export const appReducer = (state = initialState(), action) => {
     toggleSidebar,
     tutorial,
     tutorialChoice,
-    tutorialStep,
+    tutorialStep
```

Comment:
Last item should have a trailing comma as it produces a better diff when items are added or rearranged.

Reply:
Got it.